core/src/entry_verification_service/worker.rs

```diff
@@ -0,0 +1,524 @@
+//! Execution part of the [`super::EntryVerificationService`].
+
+use {
+    super::{DataSetFirstShred, SharedResults, SlotVerificationStatus},
+    crossbeam_channel::{self, RecvError, RecvTimeoutError},
+    solana_entry::entry::Entry,
+    solana_ledger::blockstore::Blockstore,
+    solana_sdk::{clock::Slot, hash::Hash},
+    std::{
+        collections::HashMap,
+        io::Error,
+        ops::RangeInclusive,
+        result::Result,
+        sync::Arc,
+        thread,
+        time::{Duration, Instant},
+    },
+};
+
+mod commands;
+
+pub(super) enum Command {
+    /// Add new data set for the specified slot.
+    AddDataSet {
+        slot: Slot,
+        shard_indices: RangeInclusive<u32>,
```

Comment:
shred* ?

Reply:
Fixed.

Seems like I'm not the only one:
```
❯ git grep -w shard | wc -l
53

❯ git grep -w shard ledger/src/shred | wc -l
39
```